swift/src/Ice/ObjectAdapter.swift

```diff
@@ -56,6 +60,14 @@ public protocol ObjectAdapter: AnyObject {
     /// the same name.
     func destroy()
 
+    /// Install a middleware in this object adapter.
+    /// - Parameter middleware: The middleware to install.
+    /// - Returns: This object adapter.
+    /// - Throws: `Ice.InitializationException` if the object adapter's dispatch pipeline has already been created. This
+    /// creation typically occurs the first time the object adapter dispatches an incoming request.
```

Comment:
Not sure if we need to document that use is not thread-safe. I.e. you shouldn't call use concurrently on the same OA.

We don't mention this "limitation" in IceRPC: 
https://github.com/icerpc/icerpc-csharp/blob/0153bc8323107da94d1fc9a360f59d737647b480/src/IceRpc/Router.cs#L117